README.md

```diff
@@ -106,7 +109,7 @@ development](https://urllib3.readthedocs.io/en/latest/sponsors.html).
 ## For Enterprise
 
 Professional support for urllib3 is available as part of the [Tidelift
-Subscription][1].  Tidelift gives software development teams a single source for
```

Comment:
Is Tidelift still around and being used for this? I thought it was effectively shuttered. Maybe I'm remembering something else?